### DISCUSSION TITLE
How do I enable View for HLSL source in Nsight Graphics' Shader Profiler?

### DISCUSSION THREAD

**Discussion by ohmaya** [o3de/o3de#16547] [Help Wanted] [answered]
Currently, I only see DXIL in the Shader Profiler > View dropdown list.

Thanks.

**Comment by ohmaya** [comment]
ENABLE_SHADER_DEBUGGING?

**Comment by galibzon** [comment, marked as answer]
https://github.com/o3de/o3de/wiki/CPU-&-GPU-Debugging-Tools
In particular:
For shaders you wish to debug, you may add the following compiler options to the .shader file.
```json
{
    "AddBuildArguments" :
    {
        "debug": true
    }
}
```
